Tasks/NuGetCommandV2/task.json

```diff
@@ -9,7 +9,7 @@
   "author": "Microsoft Corporation",
   "version": {
     "Major": 2,
-    "Minor": 252,
+    "Minor": 253,
     "Patch": 1
```

Comment:
nit: wouldn't the patch version be 0 ?

Now BuildGenConfigs are gone and versions aren't auto updated depending on the config, but DRIs seeing the patch as 0 makes it easy to see this is default and first change (no hotfix) for the task.

Reply:
But I am good to merge as is

Reply:
Dropped the patch version to 0.